### ISSUE TITLE
[Bug]: Cannot make two postgres databases publicly available simultanously

### ISSUE THREAD

**Issue by sisou** [coollabsio/coolify#3376]
### Description

I have two environments (staging and production) on the same server. Both have a Postgres database, created from Coolify's recipe (the production env was cloned from the staging env).

Earlier I needed to do a quick check on the production database and checked the "Make it publicly available" checkbox for it. It started throwing 500 errors, which I dismissed.

But when I tried connecting to the `psql` connection string, I found that it connects to the staging database.

I then noticed that I had the public checkbox checked there, too. Only when unchecked both checkboxes and then re-checked the production one, I could connect to the connection database.

**When trying to reproduce the problem**, it "correctly" auto-unchecked the checkbox in the second database after the config-update request returned the below error and didn't let me get into the state of thinking both databases are public.

### Minimal Reproduction (if possible, example repository)

Create two postgres databases (not sure if they need to be in different environments) and make both publicly available. Notice 500 errors and the second database not being accessible through the public connection string.

### Exception or Error

 noswc0c8wc8cskow80kw04sw-proxy Pulling 
 noswc0c8wc8cskow80kw04sw-proxy Pulled 
time="2024-09-10T08:42:19Z" level=warning msg="Found orphan containers ([kwc8wko-proxy coolify-proxy]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up."
 Container noswc0c8wc8cskow80kw04sw-proxy  Creating
 Container noswc0c8wc8cskow80kw04sw-proxy  Created
 Container noswc0c8wc8cskow80kw04sw-proxy  Starting
Error response from daemon: driver failed programming external connectivity on endpoint noswc0c8wc8cskow80kw04sw-proxy (97930f4b219a8c33e97f51c8b2681fb64f8ed4964cd4aa7d0b8b92e77c0e5c0c): Bind for 0.0.0.0:5432 failed: port is already allocated

### Version

v4.0.0-beta.331

### Cloud?

- [ ] Yes
- [X] No

**Comment by kmbuthia** [comment]
Hey @sisou have you already tried changing the port that one of your postgres databases uses? The coolify UI gives you a field where you can edit this. For example, one of your databases can listen on port 5433...because both can't listen on 5432 if they are deployed on the same server....the error saying "port is already allocated" hints this as well.

**Comment by sisou** [comment]
Yeah, that's the reason. The Coolify service should respond with a proper error, not a 500 Server Error. You are welcome to close this issue if I should open a new issue for this.

**Comment by andrasbacsai** [comment]
I cannot reproduce this issue. The port should be unique for each exposed database.
